Userland/Services/WebDriver/Client.cpp

```diff
@@ -345,7 +347,22 @@ Web::WebDriver::Response Client::new_window(Web::WebDriver::Parameters parameter
 {
     dbgln_if(WEBDRIVER_DEBUG, "Handling POST /session/<session_id>/window/new");
     auto session = TRY(find_session_with_id(parameters[0]));
-    return session->web_content_connection().new_window(payload);
+    auto handle = TRY(session->web_content_connection().new_window(payload));
+
+    auto timeouts = TRY(session->web_content_connection().get_timeouts());
+    auto script_timeout_ms = timeouts.as_object().get_u32("script"sv).value_or(30'000);
```

Comment:
I'm not sure that the script timeout is the best value to use here, since this isn't a script. I used this value because, previously, failing to open a window correctly would cause the script timeout to be hit while waiting for a load event.

Reply:
hmm, not sure we need to go back over IPC to get the session timeouts. Given that this is an internal implementation detail (just waiting for the socket connection), I think we can reduce observability here and just pick a smallish hard-coded timeout. 5 seconds maybe?

Reply:
Agreed. Updated to use a fixed 5 second timeout instead.